tests/integration/test_github/conftest.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import pytest
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.tools.file_monitor import FileMonitor
+from wazuh_testing.modules.integrations.event_monitors import detect_integration_start
+
+
+@pytest.fixture()
+def wait_for_github_start():
```

Comment:
Check: https://github.com/wazuh/wazuh/blob/4157_tests_migration/tests/integration/test_analysisd/conftest.py#L67

Reply:
Fixed with this [commit](https://github.com/wazuh/wazuh/pull/17621/commits/9ea1a9636f06e2cd165ec8e4c44d3e7c20e6bc7b)